app/models/pricing_rule.rb

```diff
@@ -88,7 +88,7 @@ def range_overlap
     end
 
     if overlap #rules.count > 0
-      errors.add(:min, "'From' value cannot be less than 'To' values of current rules.")
+      errors.add(:min, "'From' value must be greater than 'To' values of current rules.")
```

Comment:
```suggestion
      errors.add(:min, "The 'From' value must be greater than the 'To' values of other existing rules.")
```

Reply:
Plus, this is a good moment to use i18n and put this in locales 😄